libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/ActionScope.cs

```diff
@@ -45,12 +60,30 @@ public override async Task<DialogTurnResult> BeginDialogAsync(DialogContext dc,
             }
         }
 
+        /// <summary>
+        /// Called when the dialog is _continued_, where it is the active dialog and the
+        /// user replies with a new activity.
+        /// </summary>
+        /// <param name="dc">The <see cref="DialogContext"/> for the current turn of conversation.</param>
+        /// <param name="cancellationToken">A cancellation token that can be used by other objects
+        /// or threads to receive notice of cancellation.</param>
+        /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
         public override async Task<DialogTurnResult> ContinueDialogAsync(DialogContext dc, CancellationToken cancellationToken = default(CancellationToken))
         {
             // We're being continued after an interruption so just run next action
             return await OnNextActionAsync(dc, null, cancellationToken).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Called when a child dialog completed this turn, returning control to this dialog.
+        /// </summary>
+        /// <param name="dc">The dialog context for the current turn of the conversation.</param>
+        /// <param name="reason">Reason why the dialog resumed.</param>
```

Comment:
I think it should go a 's after dialog?
Alternatives
`Reason why the dialog's resumed.`
`Reason why the dialog is resumed.`

Reply:
I disagree with @sw-joelmut's remark, I think both the current change and the proposed alternatives are ok. (Think of q: What resumed? a:The dialog).